src/request/components/RequestDetailPanelDataOperationCommand.tsx

```diff
@@ -0,0 +1,17 @@
+import React = require('react');
+import Highlight = require('react-highlight');
+
+export interface IRequestDetailPanelDataOperationCommandProps {
+    command: string,
+    language: string
+}
+
+export class RequestDetailPanelDataOperationCommand extends React.Component<IRequestDetailPanelDataOperationCommandProps, {}> {
+    public render() {
+        return (
+            <div className='tab-data-operation-command'>
+                <Highlight language={this.props.language}>{this.props.command}</Highlight>
+            </div>
+        );
+    }
+}
```

Comment:
Do we gain much by this being a separate component instead of just being in `DataComponent.tsx`?